homeassistant/components/hive/__init__.py

```diff
@@ -54,16 +83,45 @@ class HiveSession:
 
 def setup(hass, config):
     """Set up the Hive Component."""
+
+    def heating_boost(service):
+        """Handle the service call."""
+        node_id = HiveSession.entity_lookup.get(service.data[ATTR_ENTITY_ID])
+        if not node_id:
+            # log or raise error
+            _LOGGER.error("Cannot boost entity id entered.")
+            return
+
+        minutes = service.data[ATTR_TIME_PERIOD]
+        temperature = service.data.get(ATTR_TEMPERATURE)
```

Comment:
Please use `dict[key]` for required schema keys and keys with default schema values.